kube/pod.go

```diff
@@ -110,11 +110,18 @@ func getContainerImageName(role *model.Role, settings *ExportSettings) (string,
 	if err != nil {
 		return "", err
 	}
-	imageName := builder.GetRoleDevImageName("", "", settings.Repository, role, devVersion)
-	value := "{{ default \"docker.io\" .Values.kube.registry -}}/"
-	value += "{{- .Values.kube.organization -}}/"
-	value += imageName
-	return value, nil
+
+	var imageName string
+	if settings.CreateHelmChart {
+		name := builder.GetRoleDevImageName("", "", settings.Repository, role, devVersion)
+		imageName = "{{ default \"docker.io\" .Values.kube.registry -}}/"
+		imageName += "{{- .Values.kube.organization -}}/"
+		imageName += name
```

Comment:
I wonder if you could just write:

```
registry := `{{ ... .Values.kube.registry }}`
org := `{{ .Values.kube.organization }}`
imageName = builder.GetRoleDevImageName(registry, org, ...)
```

Reply:
Hmm, would require an exception clause in `SanitizeDockerName` to do nothing if the name starts with `{{` and ends with `}}`.

Reply:
Actually, a default is still needed. :( Still thinking about that one.

Reply:
Why is a default needed, the code handles `""` correctly:

https://github.com/SUSE/fissile/blob/master/builder/role_image.go#L505-L507

Reply:
Ok, updated the sanitize function.

The image name generated by the helm template is `docker.io/splatform/foo:1234`, and *fortunately* the docker client treats that the same as `splatform/foo:1234`, so no changes are needed to the SCF project. I was worried that it would force a change to the build scripts to include `docker.io` in the image names, but that doesn't seem to be the case.